radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/forks/modules/MainnetForksModule.java

```diff
@@ -194,4 +196,35 @@ ForkBuilder olympiaRadixProtocolFork1() {
             100, // 100 max validators
             MSG.maxLength()));
   }
+
+  @ProvidesIntoSet
+  ForkBuilder olympiaShutdown() {
+    return new ForkBuilder(
+            "olympia-shutdown",
+            ImmutableSet.of(
+                new CandidateForkConfig.Threshold(
+                    (short) 7900, // 79% required
+                    10 /* for at least 10 consecutive epochs (around 6 hours) */)),
+            // Estimated (as of 2023-08-21) to be around 2023-09-27 06:00:00 UTC
+            34200L,
```

Comment:
This placeholder needs fixing for the release. I'm expecting to see something of order of magnitude ~32833